app/services/katello/cert/ca.rb

```diff
@@ -0,0 +1,21 @@
+module Katello
+  module Cert
+    module Ca
+      def self.ueber_cert(repository)
+        repository.organization.debug_cert
+      end
+
+      def self.ca_cert
+        File.open(Setting[:ssl_ca_file], 'r').read
+      end
+
+      def self.ssl_client_cert
+        OpenSSL::X509::Certificate.new(File.open(SETTINGS[:katello][:ssl_client_cert], 'r').read)
```

Comment:
I assume these settings are going to be set by the installer, so I think this code should either check for their existence or we should put 'default' paths in (https://github.com/Katello/katello/blob/master/lib/katello/engine.rb#L9). 

I would also think about changing the naming to reflect what these are, for example `[:katello][:pulp_client_cert]`